src/components/entity-panel/sections/tactic/TheTacticListContent.vue

```diff
@@ -184,35 +182,35 @@ export default class TacticSelectorContent extends Vue {
         break
     }
   }
-  tacticMenuColour (tactic: Tactic, item: TacticMenuItem) {
+  tacticMenuColour (tactic: Tactic, item: MenuItem) {
     if (item.action === TacticMenuOptions.PIN && tactic.pinned) {
       return 'primary'
     } else if (item.action === TacticMenuOptions.DELETE) {
       return 'error'
     }
     return ''
   }
-  tacticMenuIcon (tactic: Tactic, item: TacticMenuItem) {
+  tacticMenuIcon (tactic: Tactic, item: MenuItem) {
     return item.icon
   }
-  tacticMenuPinText (tactic: Tactic, item: TacticMenuItem) {
-    if (item.action === TacticMenuOptions.PIN && tactic.pinned && item.titleTwo) {
-      return item.titleTwo
+  tacticMenuPinText (tactic: Tactic, item: MenuItem) {
+    if (item.action === TacticMenuOptions.PIN && tactic.isPinned && item.titleTwo) {
+      return this.$t(item.titleTwo)
```

Comment:
If dynamic key doesn't exist in translations, what is fallback?

Reply:
The English equivalent